lib/generators/bulkrax/install_generator.rb

```diff
@@ -55,6 +55,26 @@ def add_javascripts
     end
   end
 
+  def add_ability
+    file = 'app/models/ability.rb'
```

Comment:
do we need to account for if this file doesn't exist at all?

Reply:
[Hyrax expects the file to exist](https://github.com/samvera/hyrax/blob/bf5df6f9d47ff04dc5d103b5922bcb80a9bb821c/lib/generators/hyrax/install_generator.rb#L104-L110), so I think we would be okay to as well

Reply:
Do you know of / have a case where the file doesn't exist?

Reply:
ah ok. no, I don't know of any instance where it wouldn't...just wanted to make sure our bases were covered. 🙂

Reply:
It's a completely valid question, [we just ran into that issue in the Actions repo](https://github.com/scientist-softserv/actions/commit/9aac178a68bb9d0ee5e3eccb11899b5da7025f80) last week. So, better safe than sorry 🙂